### PR TITLE
Diff synapse and ADF

### PR DESCRIPTION
**Pull request by idear1203**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json [hunk 1]
```diff
@@ -1,13 +1,24 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "ArtifactsClient",
-    "version": "2019-06-01-preview"
+    "title": "DataFactoryManagementClient",
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json:4]: pls undo such changes from ArtifactsClient to DataFactoryManagementClient across all files

Comment (ajay-arora) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json:4]: similarly the API version and securityDefinitions portion that got added

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
   },
   "paths": {},
   "definitions": {
     "DataFlow": {
-      "description": "Azure Synapse nested object which contains a flow with data movements and transformations.",
+      "description": "Azure Data Factory nested object which contains a flow with data movements and transformations.",
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json:21]: pls undo the portion which renames Synapse to Data Factory

**Review by ajay-arora — COMMENTED** [review]
Please undo all changes in artifacts.json.
Please remove the file entirely : ManagedPrivateEndpoint.json, ManagedVirtualNetwork.json

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json [hunk 3]
```diff
@@ -29,10 +40,10 @@
         },
         "linkedServiceName": {
           "description": "Linked service reference.",
-          "$ref": "../artifacts.json#/definitions/LinkedServiceReference"
+          "$ref": "../datafactory.json#/definitions/LinkedServiceReference"
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json:43]: pls undo these changes to the paths

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json [hunk 4]
```diff
@@ -0,0 +1,1537 @@
...
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "IntegrationRuntime": {
+      "description": "Azure Data Factory nested object which serves as a compute resource for activities.",
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json:21]: update to Synapse

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json [hunk 5]
```diff
@@ -0,0 +1,1537 @@
...
+    },
+    "ManagedIntegrationRuntimeTypeProperties": {
+      "description": "Managed integration runtime type properties.",
+      "type": "object",
+      "properties": {
+        "computeProperties": {
+          "description": "The compute resource for managed integration runtime.",
+          "$ref": "#/definitions/IntegrationRuntimeComputeProperties"
+        },
+        "ssisProperties": {
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json:90]: ideally this property should be removed since ssis is not supported in Synapse.

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json [hunk 6]
```diff
@@ -0,0 +1,1537 @@
...
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SelfHostedIntegrationRuntimeTypeProperties"
+        }
+      }
+    },
+    "SelfHostedIntegrationRuntimeTypeProperties": {
+      "description": "The self-hosted integration runtime properties.",
+      "type": "object",
+      "properties": {
+        "linkedInfo": {
```
Comment (ajay-arora) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json:541]: same here, this property also needs to be removed Linked IR is not supported in Synapse

**Review by ajay-arora — CHANGES_REQUESTED** [review]

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json (hunk 1 above)
Comment (idear1203) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json:4]: @ajay-arora please note that this PR is only for diff. I will need ADF team to help update the Swagger spec rather than do it by myself.
